docker-compose-develop.yml

```diff
@@ -3,15 +3,16 @@ services:
   develop:
     build: .
     ports:
-      - "3600:3600"
+      - "3602:3602"
     container_name: gfw-prodes-loss-api-develop
     environment:
-      PORT: 3600
+      PORT: 3602
       NODE_PATH: app/src
       NODE_ENV: dev
       CARTODB_USER: wri-01
       CT_URL: http://mymachine:9000
-      LOCAL_URL: http://mymachine:3600
+      CT_TOKEN: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Im1pY3Jvc2VydmljZSIsImNyZWF0ZWRBdCI6IjIwMTYtMDktMTQifQ.IRCIRm1nfIQTfda_Wb6Pg-341zhV8soAgzw7dd5HxxQ
```

Comment:
@samdavidge what is this token? Is it okay to have in source control?

Reply:
Hey @ethanroday this is a local/test key that is used (and committed) across many of the repos.